apps/client/src/features/rundown/event-block/composite/EventBlockChip.tsx

```diff
@@ -75,3 +78,51 @@ function EventUntil(props: EventUntilProps) {
     </Tooltip>
   );
 }
+
+interface EventReportProps {
+  className: string;
+  id: string;
+  duration: number;
+}
+
+function EventReport(props: EventReportProps) {
```

Comment:
I wonder if we can make this more efficient.

We know that, once an event has run, the report will never change until it runs again
Do you agree? if so, we can find a way together, to only run this once when the event finishes